practice-solutions.rb

```diff
@@ -0,0 +1,38 @@
+# def stretch(arr)
+#   list = []
+#
+#   arr.each do |a|
+#     if a % 2 == 0
+#       2.times do
+#         list << a/2
+#       end
+#     else
+#       list << (a/2) + 1
+#       list << a/2
+#     end
+#
+#   end
+#   return list
+# end
+#
```

Comment:
FYI - I like your commented out solution. The only thing I'd change is removal of the `2.times do` and just use that line inside, twice.